context-free.md

```diff
@@ -2,21 +2,61 @@
 
 ##  Who is this programming language for?
```

Comment:
Comment about Markdown: One thing I learned is that you can highlight a passage of text, use the keyboard shortcut (such as ctrl+b for bold), and Markdown will automatically bold the section of text! This also works for many other keyboard shortcuts.